test/linalg1.jl

```diff
@@ -2,7 +2,7 @@ debug = false
 
 import Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
-n = 10
+n = 10  # WARNING: n must be >= 10 for GeneralizedSchur tests
```

Comment:
Excellent, I think that will be better anyways.  I didn't want to make too many changes to the files, I will fix that.